Superalloys

WideMann opened the pull request:
This PR is about superalloys.

WideMann committed:
Superalloys added

+ fixed graphene chain ids

WideMann committed:
Superalloy crafting

bruberu commented on resources/langfiles/lang/en_us.lang:
```diff
@@ -1986,6 +1986,16 @@ susy.material.cryogenic_solder=Cryogenic Solder
+susy.material.stellite_j=Stellite J
+susy.material.stellite_six=Stellite 6
+susy.material.hsla_eighty_x=HSLA 980X
+susy.material.food_grade_stainless_steel=Food Grade Stainless Steel
```
Technically should be localized as Food-Grade Stainless Steel (with the hyphen)

WideMann committed:
Merge branch 'SymmetricDevs:master-ceu' into stuffadditions

WideMann committed:
More work

Added hyphen to FG stainless
Changed recipes to not use fluids and rebalanced them
Removed GT superalloys and their components

WideMann committed:
Merge remote-tracking branch 'origin/stuffadditions' into stuffadditions
